pages/linux/google-chrome.md

```diff
@@ -0,0 +1,12 @@
+# google-chrome
+
+> The web browser from Google.
+> More information: <https://chrome.google.com>.
+
+- Run with a temporary profile:
```

Comment:
But it's not temporary, as you can run it again with the same command (unless `path/to/chrome_dev` is deleted).

Reply:
Do you want I change the text to `Run with a custom profile directory`?

Reply:
Yep, that's good.

Reply:
Done!